### PR TITLE
Update Notifier bridge DSN in readme

### PR DESCRIPTION
**Pull request by jeremyFreeAgent**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      |no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        |

Add missing DSN documentation in README

### CODE REVIEW

**Comment by fabpot** [comment]
Why is it a draft?

File: src/Symfony/Component/Notifier/Bridge/Firebase/README.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ Firebase Notifier
 
 Provides Firebase integration for Symfony Notifier.
 
+DSN should be as follow:
+
+```
+firebase://username:password@default
```
Comment (noniagriconomie) [inline src/Symfony/Component/Notifier/Bridge/Firebase/README.md:9]:
```suggestion
firebase://USERNAME:PASSWORD@default
```

imho uppercased is better to be similar to others doc
and also we clearly see what is "configurable" from what is not :)

**Comment by fabpot** [comment]
@jeremyFreeAgent Can you finish this work? As I'm trying to cleanup PRs, I'd like to avoid keeping draft ones around for too long.
Thank you.

File: src/Symfony/Component/Notifier/Bridge/GoogleChat/README.md [hunk 2]
```diff
@@ -3,7 +3,13 @@ Google Chat Notifier
...
 Provides Google Chat integration for Symfony Notifier.
 
-    googlechat://ACCESS_KEY:ACCESS_TOKEN@default/SPACE?threadKey=THREAD_KEY
+DSN should be as follow:
+
+```
+googlechat://ACCESS_KEY:ACCESS_TOKEN@default/SPACE?threadKey=THREAD_KEY
+```
+
+`ACCESS_KEY` and `ACCESS_TOKEN` are your Google Chat authentication info ; `SPACE` is the Google Chat space ; `THREAD_KEY` is the the Google Chat message thread.
```
Comment (GromNaN) [inline src/Symfony/Component/Notifier/Bridge/GoogleChat/README.md:12]: `THREAD_KEY` is a thread identifier string that can be specified to group messages into a single thread.

https://developers.google.com/hangouts/chat/reference/rest/v1/spaces.messages/create#body.QUERY_PARAMETERS.thread_key

**Comment by jeremyFreeAgent** [comment]
I'm back from my vacations. I do it now. I split it in two PRs, one for 5.1 and the second one for 5.2.

**Comment by jeremyFreeAgent** [comment]
Closed in favour of #37962 for (5.1)

**Pull request closed without merging**